UI/Hierarchy.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#ifndef HIERARCHY_H
+#define HIERARCHY_H
+
+#include "imgui.h"
+#include <string>
+
+extern std::string selectedNode;
```

Comment:
Any reason why you need this to be extern?

Reply:
Needed to access it in the inspector, but realized a getter function would be better so changed to that